src/Estimators/src/FloatingBaseEstimator.cpp

```diff
@@ -108,26 +119,42 @@ bool FloatingBaseEstimator::advance()
         m_measPrev = m_meas;
     }
 
+    if (m_useModelInfo && m_options.kinematicsUpdateEnabled)
+    {
+        // update robot state with previous state estimates
+        if ( (m_meas.encoders.size() == m_modelComp.nrJoints()) ||
+             (m_meas.encodersSpeed.size() == m_modelComp.nrJoints()) )
+        {
+            if (!m_modelComp.kinDyn()->setRobotState(m_estimatorOut.basePose.transform(),
+                                                     iDynTree::make_span(m_meas.encoders.data(), m_meas.encoders.size()),
+                                                     m_estimatorOut.baseTwist,
+                                                     iDynTree::make_span(m_meas.encodersSpeed.data(), m_meas.encodersSpeed.size()),
+                                                     iDynTree::make_span(m_options.accelerationDueToGravity.data(), m_options.accelerationDueToGravity.size())))
```

Comment:
In theory you don't need `iDynTree::make_span`

Reply:
Removed the span in https://github.com/dic-iit/bipedal-locomotion-framework/pull/254/commits/10e3009d2a596e5fe5285718b91c776db4683086.